chia/cmds/wallet.py

```diff
@@ -48,6 +48,15 @@ def get_transaction_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: in
     show_default=True,
     required=True,
 )
+@click.option(
+    "-l",
+    "--limit",
+    help="Max number of transactions to return",
+    type=int,
+    default=1000000000,
```

Comment:
This is a pretty big change to the default from previous versions - and if I want "all" - well I guess this number is big enough for all.

Reply:
we could do default 50 like before, but i think it's reasonable to default to all. I just downloaded 36k transactions and it took like 20 seconds.

Reply:
the RPC doesn't have a way to request all, we much provide a limit. I guess we could do UINT32 max

Reply:
Yea , in PR #9447, I did add in an all option to the RPC and this then uses the existing get_all_transactions_for_wallet function (and a bunch of other changes that were invalidated by the big merge)

Reply:
Just updated that max. Your PR looks good to, i'm fine with merging that if you want to update. Otherwise I think we should merge this to fix the immediate problem of broken CLI. Many users will exceed the 50 tx limit soon.

Reply:
My PR needs a largish fix-up because it conflicts with changes already made in the merge (especially how the actual queries work). Yea, if the performance is fine, I guess I'm ok with it as just a big number - I agree we need to get this out